htdocs/user/class/user.class.php

```diff
@@ -1483,9 +1483,9 @@ function update($user, $notrigger=0, $nosyncmember=0, $nosyncmemberpass=0, $nosy
 					// This user is linked with a member, so we also update member information
 					// if this is an update.
 					$adh=new Adherent($this->db);
-					$result=$adh->fetch($this->fk_member);
+					$result=$adh->fetch('',$this->fk_member);
```

Comment:
this->fk_member is a int key.
So the fetch must be done in the first parameter.
What it is possible to do, is to test $adh->entity after the fetch, and if it is not on same entity, we ignore the update of the member.